src/common/QuirksManager.cpp

```diff
@@ -216,7 +226,8 @@ bool QuirksManager::isConversionNeeded(
     // If a SAMPLE RATE is specified for low latency then let the native code choose an optimal rate.
     // TODO There may be a problem if the devices supports low latency
     //      at a higher rate than the default.
-    if (builder.getSampleRate() != oboe::Unspecified
+    if (OboeGlobals::areWorkaroundsEnabled()
```

Comment:
Don't check areWorkaroundsEnabled() here.
We always want to convert in Oboe if the app requests Oboe SRC.
This is more of oboe feature than a bug workaround.

Reply:
Should the other two locations be changed or not?

Reply:
I changed left sample rate default as is and changed Int16 -> Float for input and output streams.